### PR TITLE
Add skip button to onboarding pages

### PR DESCRIPTION
**Pull request by CNimmo16**
Fix for issue #117 - Copied over the skip button from the landing page onto the "what is climate positive design" and map pages. Button is "sign in" or "skip" depending on authentication state on landing page, but kept as always "skip" on following pages.

### CODE REVIEW

**Comment by edanweis** [comment]
Thanks cameron, just a few things to fix

1. Skip button has a solid background on `OnBoard2.vue` but transparent on `OnBoard1.vue`. Make them all transparent.
2. Increase z-index on `<v-btn />` to 5 so it appears above the headline
3. I think we should change "skip" to say "Sign In" to avoid people thinking it "skip this step" of the onboarding.

**Comment by CNimmo16** [comment]
1 and 3 done, which of the buttons is 2 referring to? Couldn't find a page where the button is being overlapped.

File: src/views/onboarding/OnBoard2.vue [hunk 1]
```diff
@@ -436,6 +448,9 @@ catch(err) {
 <style lang="stylus">
 @import '~@/stylus/main'
 
+.sign-in-button.ma-3.on-top {
+  z-index: 3 !important;
```
Comment (edanweis) [inline src/views/onboarding/OnBoard2.vue]: Change to 5, headline on line 30 overlaps on mobile widths.

**Comment by CNimmo16** [comment]
Adjusted the header on the map page so it line breaks to avoid the button on mobile screens.

**Pull request merged**